gamemodes/base/entities/entities/lua_run.lua

```diff
@@ -6,21 +8,57 @@ ENT.DisableDuplicator = true
 
 AccessorFunc( ENT, "m_bDefaultCode", "DefaultCode" )
 
+local ALLOWED_LUA, ALLOWED_MAPS = include( "cfc_block_luarun/config.lua" )
+local CURRENT_MAP_ALLOWED = ALLOWED_MAPS[game.GetMap()]
+local LOGGER = Logger( "cfc block luarun" )
+local MD5 = util.MD5 -- Localized MD5 encoding function
```

Comment:
No need for a comment, this code speaks for itself
```suggestion
local MD5 = util.MD5
```

Reply:
👌 Corrected.
I only put that in there to help me as I've never heard of MD5 until this project.